### PR TITLE
Fix double quotes in sql queries for Python and adds success status

### PR DESCRIPTION
**Pull request by aggFTW**

### CODE REVIEW

File: remotespark/livyclientlib/sqlquery.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def _records_to_dataframe(records_text):
             raise DataFrameParseException(u"Cannot parse object as JSON: '{}'".format(strings))
 
     def _pyspark_command(self):
-        command = u'sqlContext.sql(u"""{}""").toJSON()'.format(self.query)
+        command = u'sqlContext.sql(u"""{} """[:-1]).toJSON()'.format(self.query)
```
Comment (msftristew) [inline remotespark/livyclientlib/sqlquery.py]: `[:-1]` is not necessary

File: remotespark/utils/constants.py [hunk 2]
```diff
@@ -60,15 +60,16 @@
 NOT_STARTED_SESSION_STATUS = "not_started"
 STARTING_SESSION_STATUS = "starting"
 BUSY_SESSION_STATUS = "busy"
+SUCCESS_SESSION_STATUS = "success"
```
Comment (msftristew) [inline remotespark/utils/constants.py]: Why?

Comment (msftristew) [inline remotespark/utils/constants.py]: I've spoken to @tc0312 about this and he seemed to indicate that it is a bug if interactive Livy sessions over go to success state

Comment (aggFTW) [inline remotespark/utils/constants.py]: Talked to him just now and he says a session should not be but might be momentarily in success state. He said we should add it and treat it as a final status.

**Pull request merged**